.github/workflows/release.yaml

```
@@ -0,0 +1,58 @@
+name: Release
+
+on:
+  workflow_dispatch: {}
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: true
+        fetch-depth: 0
+    - name: Set environment variables
+      run: ./build/ci/set_environment.sh
+    - name: Build chart
+      run: ./build/chart.sh
+      env:
+        IMAGE_REPOSITORY: ghcr.io/${{ github.repository }}
+    - name: Draft release
+      id: draft_release
+      uses: actions/create-release@v1
+      with:
+        release_name: ${{ env.GIT_TAG }}
+        tag_name: ${{ env.GIT_TAG }}
+        body: A MITS release.
```

```suggestion
        body: Minibroker Interegration Tests ${{ env.GIT_TAG }}.
```
We could also consider using things like [github-changelog-generator](https://github.com/github-changelog-generator/github-changelog-generator) to generate a change log body, and then use that as the release notes.  That can be later, though; I don't feel like trawling through to find one I like right now.